3차/이동건_핸드폰-번호-가리기.py

```diff
@@ -0,0 +1,11 @@
+# 23:33 ~ 23:36
+
+def create_star(num):
+    s = ""
+    for i in range(num):
+        s += "*"
+    return s
+
+def solution(phone_number):
+    num = len(phone_number) - 4
+    return create_star(num) + phone_number[num:]
```

Comment:
```suggestion

def create_star(num):
    return ''.join(['*']* num)
def solution(phone_number):
    num = len(phone_number) -4
    return ''.join(list(map(lambda s: create_star(s) + phone_number[num:],[num])))
 
def create_star(phone_number):
    return ''.join(list(map(lambda x:len(x) -4,[phone_number] ))[0] * ['*']  )
def solution(phone_number):
    num = len(phone_number) - 4
    return ''.join(list(map(lambda x:create_star(x)+phone_number[num:],[phone_number])))

 이렇게 바꿔도 되고 사실상 이건 그냥 뻘짓에 가깝구요
```

```suggestion
def create_star(num):
    return ''.join(['*'] * num)
def solution(phone_number):
    num = len(phone_number) - 4
    return create_star(num) + phone_number[num:]
```
이렇게 바꾸니 성능이 조금더 좋아지네요

바꾸기전

![image](https://user-images.githubusercontent.com/24598628/93233052-9938d000-f7b5-11ea-869a-9f3a2eadf98d.png)


바꾸기 후

![image](https://user-images.githubusercontent.com/24598628/93233021-8c1be100-f7b5-11ea-99d1-2e135c861c88.png)